test/test_transformers.py

```diff
@@ -1899,6 +1899,16 @@ def test_flash_attention_fail_with_non_square_causal_attention(self, device):
                 self.assertRaises(RuntimeError, lambda: torch.nn.functional.scaled_dot_product_attention(
                     q, k, v, None, 0.0, is_causal=True))
 
+    @onlyCUDA
+    def test_mem_eff_attention_fail_with_batch_size_geq_65536(self, device):
+          query = torch.rand([2**16, 8, 4, 64], device='cuda', dtype=torch.float16)
```

Comment:
I wonder if this will OOm on runners?

Reply:
The spatial dims could be dialed down. Is this a temporary fix, or the user would need to split tensors?